analyzers/src/SonarAnalyzer.Utilities/Rules.Description/S107_cs.html

```diff
@@ -15,4 +15,22 @@ <h2>Compliant Solution</h2>
 ...
 }
 </pre>
+<h2>Exceptions</h2>
+<pre>
+public class BaseClass
+{
+	public BaseClass(int param1)
+	{
+	...
+	}
+}
+
+public class ExtensionClass : BaseClass
```

Comment:
here also

Reply:
This directory is copy/paste of the other one, it will be fixed together